lib/pinyin.ex

```diff
@@ -100,28 +100,60 @@ defmodule Pinyin do
   ## Examples
 
       iex> Pinyin.from_marked("nǐ")
-      %Pinyin{tone: 3, word: "ni"}
+      %Pinyin{tone: 3, final: "i", initial: "n"}
```

Comment:
I initially did that but noticed in the iex it would show up in alphabetic order. (Yes, tone wouldn't be in the correct order either..)
Since the test doesn't seem to care about this and it's documentation, I can definitely make the order more readable.

Reply:
Done.